src/coreclr/jit/simdashwintrinsic.cpp

```diff
@@ -641,19 +655,6 @@ GenTree* Compiler::impSimdAsHWIntrinsicSpecial(NamedIntrinsic       intrinsic,
             break;
         }
 
-        case NI_VectorT128_ToScalar:
```

Comment:
This I believe is just for TP reason, right?

Reply:
Its unnecessary since we're handling decomposition for `ToScalar` vs `GetElement(0)` (since it has to be lowered to `GetElement(0), GetElement(1)`)